spec/tasks/plugin_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+RSpec.describe "Plugin rake tasks" do
+  before do
+    Rake::Task.clear
+    Discourse::Application.load_tasks
+    STDOUT.stubs(:write)
+  end
+
+  describe "plugin:create" do
+    def invoke(plugin_name)
+      error = nil
+      stdout =
+        capture_stdout do
+          Rake::Task["plugin:create"].invoke(plugin_name)
```

Comment:
We don't really want our test suite to be dependent on a network connection... which makes testing this quite tricky. 

The `discourse_theme` CLI solves this by having a [super lightweight copy](https://github.com/discourse/discourse_theme/tree/main/test/fixtures/skeleton-lite) of the skeleton. Maybe we do that here as well, and stub out the `git clone` call to just be a `cp` call?

Or... tbh... maybe we can just skip testing this for now. It's not 'production' code, and the other `plugin:*` rake tasks don't have any tests.